src/msg/async/AsyncMessenger.cc

```diff
@@ -291,7 +291,15 @@ void AsyncMessenger::ready()
 {
   ldout(cct,10) << __func__ << " " << get_myaddr() << dendl;
 
-  stack->start();
+  stack->ready();
+  if (pending_bind) {
```

Comment:
no, we don't want to introduce odd bind logic here, we need to maintain bind request in RDMAStack side.